README.md

```diff
@@ -23,84 +23,11 @@ pod 'FTPropertyWrappers', '~> 1.0'
 
 The main aim of this package is to provide programmer with access to commonly used features or snippets with as easy API as possible. Runtime efficiency, while important, is not the main focus of this package. As for today, this package contains wrappers for following features:
 
-- `UserDefaults` for storing a value inside Foundation's User Defaults
-- `StoredSubject` which is simple multidelegate/observing primitive for Swift
```

Comment:
Kdyz tyto objekty byly nahrazeny, nemely byt zde zmineny ty nove tedy?

Reply:
Nene, `CurrentValueSubject` je přímo součástí `Combine` a `AppStorage` zase `SwiftUI`.

Reply:
Aha, pak jsem asi nepochopila, proc se tu zminovalo UserDefaults :D to je soucasti Foundation, coz mi prijde taky jako zaklad